### PR TITLE
EWWW missing binaries

### PR DESCRIPTION
**Pull request by wilson1000**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -31,7 +31,25 @@ RUN curl -fL -o imagick.tgz 'https://pecl.php.net/get/imagick-3.7.0.tgz'; \
...
-    rm -rf imagick.tgz /tmp/imagick-3.7.0;
+    rm -rf imagick.tgz /tmp/imagick-3.7.0
+
+# https://docs.ewww.io/article/6-the-plugin-says-i-m-missing-something
+ARG GIF=1.95
+ARG JPG='9f'
+ARG PNG=0.7.8
+ARG WEBP=1.4.0
+
+RUN curl -fL -o gifsicle.tar.gz "https://www.lcdf.org/gifsicle/gifsicle-${GIF}.tar.gz"; \
```
Comment (earthlingdavey) [inline Dockerfile]: I know appearances aren't everything, but https://www.lcdf.org doesn't look like a super professional/secure site.

Would downloading from the GitHub release https://github.com/kohler/gifsicle still work here?

**Review by earthlingdavey — DISMISSED** [review]
Looks all good, just a comment concerning of the source file's security.

File: Dockerfile (hunk 1 above)
Comment (wilson1000) [reply Dockerfile]: Thanks for pointing that out. A little dig uncovered that Eddie Kohler is a professor at Harvard: https://www.read.seas.harvard.edu/~kohler/

He created the site to house the programme, I'm thinking he's more a programmer than a designer lol :)

Comment (wilson1000) [reply Dockerfile]: To answer your question; I'm sure getting the tarball from GitHub would work and the program makes use of the boilerplate install method and the tarball can be accessed here: 
https://github.com/kohler/gifsicle/archive/refs/tags/v1.95.tar.gz

In your opinion @EarthlingDavey - would it be more secure to grab it from here?

Comment (earthlingdavey) [reply Dockerfile]: Probably a little bit, it would have higher visibility and maybe be more auditable in case a security issue did come up.

Comment (earthlingdavey) [reply Dockerfile]: > he's more a programmer than a designer lol

💯😁

**Review by earthlingdavey — APPROVED** [review]
Looks good :)

**Pull request merged**